webapp/src/app/components/leaderboard/leaderboard.stories.ts

```diff
@@ -0,0 +1,18 @@
+import { type Meta, type StoryObj } from '@storybook/angular';
+import { LeaderboardComponent } from './leaderboard.component';
+
+const meta: Meta<LeaderboardComponent> = {
+  title: 'Components/Leaderboard',
+  component: LeaderboardComponent,
+  tags: ['autodocs']
```

Comment:
I think we would somehow inject a provider for a mocked `LeaderboardService` that returns the mock data here. Alternatively, we would dumb down the leaderboard component into for example `LeaderboardComponent` (smart with service) and `LeaderboardTableComponent` (dumb no service).